sdk/loadtesting/load-testing-rest/src/parameters.ts

```diff
@@ -78,42 +94,34 @@ export type TestUploadFileParameters = TestUploadFileQueryParam &
   RequestParameters;
 export type TestGetFileParameters = RequestParameters;
 export type TestDeleteFileParameters = RequestParameters;
-
-export interface TestListFilesQueryParamProperties {
-  /** Continuation token to get the next page of response */
-  continuationToken?: string;
```

Comment:
swagger change introduced: https://github.com/Azure/azure-rest-api-specs/pull/21986/files. not breaking